modules/_labs/pagination/react/lib/Pagination/Controls.tsx

```diff
@@ -0,0 +1,144 @@
+import * as React from 'react';
+import {IconButton, IconButtonProps} from '@workday/canvas-kit-react-button';
+import {
+  chevronLeftSmallIcon,
+  chevron2xLeftSmallIcon,
+  chevronRightSmallIcon,
+  chevron2xRightSmallIcon,
+} from '@workday/canvas-system-icons-web';
+
+import {PaginationModel} from './types';
+import {HStack} from './common/Stack';
+import {useRTL} from './common/utils/useRTL';
+
+export type ControlButtonProps = PaginationModel & IconButtonProps;
+
+export type ControlsProps = React.HTMLAttributes<HTMLDivElement>;
+
+export const Controls = ({children, ...elemProps}: ControlsProps) => {
+  return (
+    <HStack spacing="xxxs" alignItems="center" {...elemProps}>
+      {children}
+    </HStack>
+  );
+};
+
+export const JumpToFirstButton = ({
+  'aria-label': ariaLabel,
+  state,
+  events,
+  onClick,
+  ...restProps
+}: ControlButtonProps) => {
+  const isDisabled = state.currentPage <= state.firstPage;
+  const handleClick = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    if (isDisabled) {
+      return;
+    }
+    onClick?.(e);
+    events.setCurrentPage(state.firstPage);
+  };
+  const {shouldUseRTL} = useRTL();
+  const icon = shouldUseRTL ? chevron2xRightSmallIcon : chevron2xLeftSmallIcon;
+  return (
+    <IconButton
+      aria-disabled={isDisabled || undefined}
+      aria-label={ariaLabel}
```

Comment:
I've been letting these fall back to `...restProps`. The effect is the same, but less code.

Reply:
I'm good with that. Previously, we opted to be more explicit, but I also like less code.

Reply:
I think we can be explicit with the prop definitions (especially with the JSDoc) of it, but implicitly allow the props to be applied.